.github/workflows/deploy-preview.yml

```diff
@@ -12,24 +60,12 @@ on:
       event_number:
         required: true
         type: string
-      title:
```

Comment:
Am I correct in assuming we will need to update each workflow file, and that there is not a way to support both? Could the original inputs be accepted, and the `sources` overrides them? Bad idea?

Reply:
That might be the best way to roll this out actually.

The other thought was using one of our other branches (like this one or `robbymilo/deploy-preview`) so that we can kind of blue/green deploy changes.

In this case, I can probably modify the script to support both so we can smoothly migrate from the old to the new.

Will give you a ping after I've done that.